app/uk/gov/hmrc/transitmovements/controllers/MovementsController.scala

```diff
@@ -160,7 +166,7 @@ class MovementsController @Inject() (
       .asPresentation
       .fold[Result](
         baseError => Status(baseError.code.statusCode)(Json.toJson(baseError)),
-        movements => if (movements.isEmpty) NotFound else Ok(Json.toJson(movements))
+        movements => { println(Json.toJson(movements)); if (movements.isEmpty) NotFound else Ok(Json.toJson(movements)) }
```

Comment:
println?

Reply:
whoops - good spot, it was for debugging